src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -0,0 +1,225 @@
+
+        // backButton
+        this.add(this.backButton);
+        this.backButton.setAlignmentX(CENTER_ALIGNMENT);
+        this.add(Box.createRigidArea(new Dimension(0,VERTICAL_GAP)));
+        // Back from splitting bills event
+        this.addEntryButton.addActionListener((e -> this.backFromSplit()));
+        // backButton should be disabled at the beginning
+        this.backButton.setEnabled(false);
+
+        // addEntryButton
+        this.add(this.addEntryButton);
+        this.addEntryButton.setAlignmentX(CENTER_ALIGNMENT);
+        this.add(Box.createRigidArea(new Dimension(0,VERTICAL_GAP)));
+        // Add entry event
+        this.addEntryButton.addActionListener((e -> this.addEntry()));
+        // addEntryButton should be disabled at the beginning
+        this.addEntryButton.setEnabled(false);
+
+        // deleteEntryButton
+        this.add(this.deleteEntryButton);
+        this.deleteEntryButton.setAlignmentX(CENTER_ALIGNMENT);
+        this.add(Box.createRigidArea(new Dimension(0,VERTICAL_GAP)));
+        // Delete entry event
+        this.addEntryButton.addActionListener((e -> this.deleteEntry()));
+        // deleteEntryButton should be disabled at the beginning
+        this.deleteEntryButton.setEnabled(false);
+
+        // statisticsTextArea
+        this.add(this.statisticsTextArea);
+        this.statisticsTextArea.setAlignmentX(CENTER_ALIGNMENT);
+    }
+
+    private void initBorder() {
+        TitledBorder actionBorder = new CustomTitleBorder("Actions");
+        this.setBorder(actionBorder);
+    }
+
+    private void signIn() {
+        // Get the username and password from user
+        String userName = this.usernameField.getText();
+        String userPassword = Arrays.toString(this.passwordField.getPassword());
+        System.out.println("Name: " + userName);
+        System.out.println("Password: " + userPassword);
+
+        // TODO: Call the controller of UserJoinUseCase
+
+        // If the user has successfully signed in,
+        if (this.checkUsername() & this.checkPassword()) {
+            // disable the signInButton, and enable the signOutButton and addEntryButton
+            this.signInButton.setEnabled(false);
+            this.signOutButton.setEnabled(true);
+            this.addEntryButton.setEnabled(true);
+
+            // the usernameField and passwordField shouldn't be editable
+            this.usernameField.setEditable(false);
+            this.passwordField.setEditable(false);
+
+            // enable importMenu
+            TopMenuBar tmb = (TopMenuBar) this.getRootPane().getJMenuBar();
+            tmb.getImportMenu().setEnabled(true);
+
+            // enable billTable
+            MainFrame mf = (MainFrame) SwingUtilities.getWindowAncestor(this);
+            BillTable bt = (BillTable) mf.getBillPanel().getBillTable();
+            bt.setVisible(true);
+            bt.setEnabled(true);
+        }
+    }
+
+    private boolean checkUsername() {
```

Comment:
I thought according to 
<img width="685" alt="image" src="https://user-images.githubusercontent.com/8016180/201498253-07655c15-ef74-41fb-afca-8faea3158514.png">

It was already done? 

If it will be done in future Pull Requests, consider marking it as "TODO:" so that other contributors can easily Ctrl + F to find it :3

Reply:
How to mark it as "TODO:"?